api_migration.md

```diff
@@ -0,0 +1,121 @@
+# Api Migration
+
+New API can be found in [json format here](./api.json)
+
+## /sectors
+/sectors
+/sectors/:id
+
+/sectors/:id/feedback
+/sectors/:id/visits
+/sectors/:id/activities
+
+## /regions
+/regions
+/regions/:id
+`GET` replaces `/regions/{region}/organisations`  ⏱ volunteer app
+
+## /organisations
+/organisations
+/organisations/:id || me
+`GET` replaces /api/users/cb-name 👣 visitor app 
+
+`PUT` replaces `/api/cb/details/update` 👣 visitor app
+
+### /organisations/:id/visit_activities
+`GET` replaces `/api/activities/today || all` 👣 visitor app // filter used for individual day
+
+`POST` replaces `/api/activity/add` 👣 visitor app 
+
+`PUT` replaces `/api/activity/update` 👣 visitor app 
+
+`DELETE` replaces `/api/activity/delete` 👣 visitor app 
+
+### /organisations/:id/feedback
+`GET` replaces `/api/cb/feedback` 👣 visitor app 
+
+`POST` replaces `/api/cb/feedback` 👣 visitor app 
+
+### /organisations/:id/users
+`GET` replaces `/api/users/all` 👣 visitor app   
+`GET` replaces `/api/visitors/filtered` 👣 visitor app   
+`GET` replaces `/api/visitors/all` 👣 visitor app  
+`GET` replaces `/volunteers/organisation/{organisation}` ⏱ volunteer app  
+`GET` replaces `/volunteers/organisation360/{organisation}` ⏱ volunteer app  
+
+### /organisations/:id/visits
+`GET` replaces `/api/users/filtered`  👣 visitor app   
+`GET` replaces `/api/users/chart-all`  👣 visitor app  
+
+### /organisations/:id/volunteers 
+🤔 volunteers here reffers to volunteer logs. In two minds whether this is a better category over `/volunteer_logs`  
+`GET` replaces `/organisations/{organisation}/summary` ⏱ volunteer app  
+`GET` replaces `/logs/admin/{user}` ⏱ volunteer app  
+
+/organisations/:id/meetings (🤔 i don't think we need this)
+
+### /organisations/:id/outreach
+`GET` replaces `/outreaches/{organisation}/bytype/{id}` ⏱ volunteer app
+
+## /users (does this want to be broken down for frontline?)
+/users
+### /users/register
+`POST` replaces `/api/cb/register` 👣 visitor app
+
+### /users/login
+`POST` replaces `/api/cb/login` 👣 visitor app  
+`POST` replaces `/api/admin/login` 👣 visitor app  
+`POST` replaces `/users/login`  ⏱ volunteer app  
+
+### /users/:id || me
+`GET` replaces `/api/user/details` 👣 visitor app  
+`GET` replaces `/api/admin/check` 👣 visitor app  
+`GET` replaces `/api/user/name-from-scan` 👣 visitor app   🤔 not sure if this should be covered here with qr params or under a `/users/:id/qr_code` route.  
```

Comment:
I think in my old design I was using a `/search` endpoint where you can search for entities by attributes. See https://github.com/TwinePlatform/twine-visitor/blob/master/docs/new_api.md#post-apivisitorssearch